server/src/main/java/com/cloud/network/element/ConfigDriveNetworkElement.java

```diff
@@ -590,7 +599,35 @@ private boolean deleteConfigDriveIsoOnHostCache(final VirtualMachine vm, final L
         return true;
     }
 
-    private boolean createConfigDriveIso(VirtualMachineProfile profile, DeployDestination dest, DiskTO disk) throws ResourceUnavailableException {
+    private List<NicProfile> getNicProfilesForNic(VirtualMachine vm, List<? extends Nic> nics) {
```

Comment:
not sure if this is helpful but NetworkOrchestrator has already this method 

public List<NicProfile> getNicProfiles(final VirtualMachine vm) {

Reply:
Let me change to `getNicProfiles`